spacemacs/packages.el

```diff
@@ -3041,16 +3041,20 @@ It is a string holding:
       (sp-pair "[" nil :post-handlers
                '(:add (spacemacs/smartparens-pair-newline-and-indent "RET"))))))
 
+(defvar spacemacs-smooth-scroll-margin 5
+  "Default number of lines above/below the cursor to keep visible
+while scrolling.")
+
 (defun spacemacs/init-smooth-scrolling ()
   (if dotspacemacs-smooth-scrolling
       (use-package smooth-scrolling
         :init
-        (setq smooth-scroll-margin 5
+        (setq smooth-scroll-margin spacemacs-smooth-scroll-margin
```

Comment:
Should be a dotspacemacs variable named `dotspacemacs-smooth-scroll-margin`
A default value must be provide in `core/core-dotspacemacs.el` as well as in the template `core/templates/.spacemacs.template`.